app/frontend/pages/RunningExperiment/RunningExperiment.tsx

```diff
@@ -91,7 +91,7 @@ const NewExperiment = () => {
   }
   return (
     <StudyProvider
-      schema={{ ...schema, group: assignment.assignment.group } as SchemaV0}
+      schema={{ ...schema, group: assignment.assignment.group, pdf_interactions: []} as unknown as SchemaV0}
```

Comment:
I think this line of code should be:

```
pdf_interactions: (schema.pdf_interactions || []) as unknown
```

Reply:
The reason is that if the `schema` already has `pdf_interactions` defined in it, we should use the one in the backend rather than create a new one afresh.